annotator/src/main/java/org/cbioportal/models/MutationRecord.java

```diff
@@ -28,500 +28,87 @@
+            i++;
         }
-        return sortedAdditionalPropertiesKeys;
     }
 
-    public List<String> getSortedAdditionalPropertiesValues() {
-        List<String> sortedValues = new ArrayList<>();
-        for (String additionalPropertiesKey : getSortedAdditionalPropertiesKeys()) {
-            if (!getHeader().contains(additionalPropertiesKey)) {
-                sortedValues.add(getAdditionalProperties().getOrDefault(additionalPropertiesKey,""));
-            }
+    public String get(String header) {
+        Integer index = headers.get(header);
+        if (index == null) {
+            return "";
         }
-        return sortedValues;
+        return line.substring(beginIndices[index], endIndices[index]);
     }
- 
-    public List<String> getHeader() {
-        return header;
+
+    public GenomicLocation asGenomicLocation() {
+        GenomicLocation genomicLocation = new GenomicLocation();
+        genomicLocation.setChromosome(get(Chromosome));
+        genomicLocation.setStart(Integer.valueOf(get(Start_Position)));
+        genomicLocation.setEnd(Integer.valueOf(get(End_Position)));
+        String referenceAllele = get(Reference_Allele);
+        genomicLocation.setReferenceAllele(referenceAllele);
+        genomicLocation.setVariantAllele(MafUtil.resolveTumorSeqAllele(referenceAllele, get(Tumor_Seq_Allele1), get(Tumor_Seq_Allele2)));
+        return genomicLocation;
     }
 
-    public void initHeader() {
-        header.add("Hugo_Symbol");
-        header.add("Entrez_Gene_Id");
-        header.add("Center");
-        header.add("NCBI_Build");
-        header.add("Chromosome");
-        header.add("Start_Position");
-        header.add("End_Position");
-        header.add("Strand");
-        header.add("Variant_Classification");
-        header.add("Variant_Type");
-        header.add("Reference_Allele");
-        header.add("Tumor_Seq_Allele1");
-        header.add("Tumor_Seq_Allele2");
-        header.add("dbSNP_RS");
-        header.add("dbSNP_Val_Status");
-        header.add("Tumor_Sample_Barcode");
-        header.add("Matched_Norm_Sample_Barcode");
-        header.add("Match_Norm_Seq_Allele1");
-        header.add("Match_Norm_Seq_Allele2");
-        header.add("Tumor_Validation_Allele1");
-        header.add("Tumor_Validation_Allele2");
-        header.add("Match_Norm_Validation_Allele1");
-        header.add("Match_Norm_Validation_Allele2");
-        header.add("Verification_Status");
-        header.add("Validation_Status");
-        header.add("Mutation_Status");
-        header.add("Sequencing_Phase");
-        header.add("Sequence_Source");
-        header.add("Validation_Method");
-        header.add("Score");
-        header.add("BAM_File");
-        header.add("Sequencer");
-        header.add("t_ref_count");
-        header.add("t_alt_count");
-        header.add("n_ref_count");
-        header.add("n_alt_count");
+    public Collection<String> getHeaders() {
+        return headers.keySet();
```

Comment:
same comment as above .. this seems to present the headers in an arbitrary order.